config/secrets/sc-secrets.yaml

```diff
@@ -51,6 +51,12 @@ opensearch:
   curatorPassword: somelongsecret
   snapshotterPassword: somelongsecret
   metricsExporterPassword: somelongsecret
+  extraUsers: []
+  #    - username: static-user
+  #      definition:
+  #        password: somelongsecret
+  #        opendistro_security_roles:
+  #        - kibana_user #make sure the role exists or is created using extraRoles
```

Comment:
Maybe add `attributes` and `backend_roles` as a part of your example.
https://opensearch.org/docs/1.2/security-plugin/access-control/api/#create-user